_posts/en/posts/2016-01-24-clarification.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Clarifying Communications
+name: clarifying-communications
+type: post
+permalink: /en/2015/01/24/clarification
+share: true
+---
+The bitcoincore.org website is the official website for the Bitcoin Core project. Other sites may also host information about Bitcoin Core, but they are not under the control of the Bitcoin Core project. We know that the Bitcoin ecosystem can be confusing, and we are working hard to make these relationships more clear.
+
+For development work, Bitcoin Core mainly uses the `#bitcoin-core-dev` IRC channel on Freenode, [Github](https://github.com/bitcoin/bitcoin) and [the bitcoin-dev mailing list](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/). We also recently created a community Slack group (request an invitation [here](https://slack.bitcoincore.org)) to enable the greater Bitcoin community to connect with Bitcoin Core developers and contributors more easily.
```

Comment:
It is run primarily by core contributors, though... Bitcoin Core doesn't run anything, just it's contributors.

On January 26, 2016 9:41:51 PM PST, "David A. Harding" <notifications@github.com> wrote:
>> @@ -0,0 +1,14 @@
>> +---
>> +title: Clarifying Communications
>> +name: clarifying-communications
>> +type: post
>> +permalink: /en/2015/01/24/clarification
>> +share: true
>> +---
>> +The bitcoincore.org website is the official website for the Bitcoin
>Core project. Other sites may also host information about Bitcoin Core,
>but they are not under the control of the Bitcoin Core project. We know
>that the Bitcoin ecosystem can be confusing, and we are working hard to
>make these relationships more clear.
>> +
>> +For development work, Bitcoin Core mainly uses the
>`#bitcoin-core-dev` IRC channel on Freenode,
>[Github](https://github.com/bitcoin/bitcoin) and [the bitcoin-dev
>mailing list](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/).
>We also recently created a community Slack group (request an invitation
>[here](https://slack.bitcoincore.org)) to enable the greater Bitcoin
>community to connect with Bitcoin Core developers and contributors more
>easily.
>
>By that same standard, neither is Freenode or GitHub or Slack.  I think
>it's ok to leave these things unsaid.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/bitcoin-core/website/pull/79/files#r50943593